src/styles/blocks/product.scss

```
@@ -0,0 +1,137 @@
+.product {
+  &--catalog {
+    padding-inline: 40px;
+    display: grid;
+    grid-template-rows: 2;
+    grid-template-columns: repeat(1, 200px);
+    row-gap: 46px;
+    column-gap: 48px;
+    justify-content: center;
+
+    @media (min-width: 488px) {
+      grid-template-columns: repeat(2, 200px);
+    }
+
+    @media (min-width: 768px) {
+      grid-template-columns: repeat(3, 200px);
+    }
+
+    @media (min-width: 1024px) {
+      grid-template-columns: repeat(4, 200px);
+    }
+  }
+
+  &__card {
+    padding: 32px 16px 16px;
+    display: flex;
+    width: 166px;
```

and don't forget about box-sizing at `*` selector
```suggestion
    width: 200px;
```